### PR TITLE
Log RAW and TEXT writes

### PR DESCRIPTION
**Pull request by tronikos**

### CODE REVIEW

File: components/econet/econet.cpp [hunk 1]
```diff
@@ -226,13 +226,25 @@ void Econet::parse_message_(bool is_tx) {
...
+              ESP_LOGI(TAG, "  Unexpected Write Data Length");
+            }
+            break;
+          case EconetDatapointType::RAW:
+            ESP_LOGI(TAG, "  %s: %s", item_name.c_str(), format_hex_pretty(pdata + 14, data_len - 14).c_str());
+            break;
+          case EconetDatapointType::TEXT:
+            ESP_LOGW(TAG, "(Please file an issue with the following line to add support for TEXT)");
+            ESP_LOGW(TAG, "  %s: %s", item_name.c_str(), format_hex_pretty(pdata, data_len).c_str());
+            break;
```
Comment (barndawgie) [inline components/econet/econet.cpp:247]: There's a bunch of integer constants in here, some of them getting used repeatedly. Should these have const int variables defined for them with some reasonable names to make things more readable?

File: components/econet/econet.cpp [hunk 2]
```diff
@@ -13,14 +13,18 @@ static const uint8_t SRC_ADR_POS = 5;
 static const uint8_t LEN_POS = 10;
 static const uint8_t COMMAND_POS = 13;
 
+static const uint8_t OBJ_NAME_POS = 6;
+static const uint8_t OBJ_NAME_SIZE = 8;
+static const uint8_t FLOAT_SIZE = 4;
```
Comment (barndawgie) [inline components/econet/econet.cpp:19]: Bit of a nit, but rather than setting this as a variable I think it's cleaner to call `sizeof(float)`.

**Review by barndawgie — DISMISSED** [review]
I put one comment, but otherwise this change LGTM.

File: components/econet/econet.cpp (hunk 2 above)
Comment (tronikos) [reply components/econet/econet.cpp:19]: They are different things though. sizeof(float) depends on the architecture of the device that this code is compiled for. But this size of 4 corresponds to the float size of the econet MCU and how it represents the data in its protocol.

Comment (tronikos) [reply components/econet/econet.cpp]: Done.

**Review by barndawgie — APPROVED** [review]

**Pull request merged**